### PR TITLE
made freelancer form reusable, added props, added freelancerhomepage

### PR DESCRIPTION
**Pull request by pavlo-chernykh**
What was done:
because of the new task to make freelancer profile page, that can be editable (so it should be a form), there's a need to reuse the form freelancer-profile-questions.
In order to do that the form was moved to libs, two forms combined into one to make edit possible.
Freelancer page is created:
image
image

As for now mock user is used
education/work/skills/category is also displayed correctly (update 5/12/2022)

### CODE REVIEW

File: apps/freelance/src/pages/FreelancerEditProfile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'react-i18next';
+import { AppBar, AvatarUpload, FreelancerForm } from '@freelance/components';
+import { mockUserData } from '@freelance/components';
+
+import * as St from './styles';
```
Comment (lyudad) [inline apps/freelance/src/pages/FreelancerEditProfile/index.tsx:10]: где пробел? проверь настройки линта

File: apps/freelance/src/pages/FreelancerEditProfile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const FreelancerProfile = () => {
+  const { t } = useTranslation();
```
Comment (lyudad) [inline apps/freelance/src/pages/FreelancerEditProfile/index.tsx:13]: тоже самое 
Снимок экрана 2022-12-04 в 20 00 39

**Review by lyudad — APPROVED** [review]

File: apps/freelance/src/pages/FreelancerEditProfile/index.tsx (hunk 1 above)
Comment (pavlo-chernykh) [reply apps/freelance/src/pages/FreelancerEditProfile/index.tsx:10]: проверил, изменил. Пофиксил правило для return (раньше был вар стейтмент который мы нигде не используем), добавил правило для паддинга после импорта
image

File: apps/freelance/src/pages/FreelancerEditProfile/index.tsx (hunk 2 above)
Comment (pavlo-chernykh) [reply apps/freelance/src/pages/FreelancerEditProfile/index.tsx:13]: пофиксил

**Review by lyudad — APPROVED** [review]

**Pull request merged**